### ISSUE TITLE
🐛 Bug: Too many requests are done when copying a file with webdav

### ISSUE THREAD

**Issue by Haskaris** [tracim/tracim#6068]
### Tracim version

v4.4.3 build_097

### Issue description

When copying/moving a file/folder using webdav, the server CPU is overloaded.
With a simple HTML-document (some comments and some revisions (<30)) I can go to 1 000 API requests

### How to reproduce

1. Create a HTML-document
2. Add some revisions
3. Add some commentaries
4. Edit some of your commentaries
5. Add some mentions (optional)
6. Copy your HTML-document using webdav

### What browsers are you seeing the problem on?

Chrome, Firefox

### What devices are you seeing the problem on?

Desktop

### Related PR

PR #6069

**Comment by PhilippeAccorsi** [comment]
Do you know from which version of tracim, this kind of problem exists ?

**Comment by Haskaris** [comment]
There is multiples reasons of this CPU overload
We do some API calls for certain TLMs. We have found that we do X times 1 API call, with X being the number of app that have commentaries available, for each commentary modified. And we found that we replay the entire history of a duplicated (copy) content.

The issue with the X times 1 API call by edited comment is there since #5846 (v4.3.2 build_092)
The issue about every TLMs sent on copy, I don't know, probably forever
